build.sbt

```
@@ -1,23 +1,23 @@
 ThisBuild / tlBaseVersion := "0.7"
 ThisBuild / organization := "io.chrisdavenport"
 ThisBuild / organizationName := "Christopher Davenport"
+ThisBuild / startYear := Some(2018)
 ThisBuild / licenses := Seq(License.MIT)
 ThisBuild / developers := List(
   tlGitHubDev("christopherdavenport", "Christopher Davenport")
 )
 ThisBuild / tlCiReleaseBranches := Seq("main")
-ThisBuild / tlSonatypeUseLegacyHost := true
+ThisBuild / sonatypeCredentialHost := xerial.sbt.Sonatype.sonatypeLegacy
 
-ThisBuild / crossScalaVersions := Seq("2.12.17", "2.13.8", "3.2.2")
-ThisBuild / scalaVersion := "3.2.2"
+ThisBuild / crossScalaVersions := Seq("2.12.20", "2.13.16", "3.3.4")
```

Scala 3.3.5 is out now and used [elsewhere](https://github.com/typelevel/cats/blob/cf97ee77f794f360984f87d98efbdf155cfc995f/build.sbt#L19).
```suggestion
ThisBuild / crossScalaVersions := Seq("2.12.20", "2.13.16", "3.3.5")
```

---

We can't update to Scala 3.3.5 because that would force us on to Scala Native 0.5.x and we can't make that bump until cats-effect moves forward there